test/test_torch.py

```diff
@@ -261,7 +261,13 @@ def test_linear_algebra_scalar_raises(self):
         self.assertRaises(RuntimeError, lambda: torch.addr(m, v, s))
         self.assertRaises(RuntimeError, lambda: torch.addr(m, s, v))
 
-    def _test_math(self, torchfn, mathfn, input=None):
+    def _test_math(self, torchfn, mathfn, input=None, nonzero=False, offset=0):
+        def proc(input):
```

Comment:
What does the name proc mean?

Reply:
"process" or "preprocess". I can choose a longer name.